### PR TITLE
Meeting history dev

### PR DESCRIPTION
**Pull request by vivianho18**
Create html page to display meeting history and edit home page button to link to this page.

### CODE REVIEW

File: prototype/src/main/webapp/meeting-history.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<html>
+<head>
+  <title>GoMeet Meeting History</title>
+  <script src="script.js"></script>
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 style="text-align:center;">GoMeet Meeting History</h1>
+    <ul>
+        <h3><a href="view-meeting.html">Team Lunch</a></h3>
```
Comment (anna-giang) [inline prototype/src/main/webapp/meeting-history.html]: Nit: 2 spaces for indentation, but should be fine since this is just a prototype 👍

**Comment by anna-giang** [comment]
lgtm 😊

**Review by theresenieva — COMMENTED** [review]
lgtm

File: prototype/src/main/webapp/meeting-history.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>GoMeet Meeting History</title>
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 style="text-align:center;">GoMeet Meeting History</h1>
+    <ul>
```
Comment (ghost) [inline prototype/src/main/webapp/meeting-history.html]: This `ul` element is a little weird - normally `ul` should only have `li` elements as children. If you like the 'indented' look of this, I would replace this with a `div` and add a `margin-left: 40px` or similar style.

File: prototype/src/main/webapp/meeting-history.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (ghost) [inline prototype/src/main/webapp/meeting-history.html]: nit: this is indented weirdly - you'd want the opening and closing head tags to be on the same level.

**Review by ghost — APPROVED** [review]
LGTM, small style comments only.

File: prototype/src/main/webapp/meeting-history.html (hunk 2 above)
Comment (vivianho18) [reply prototype/src/main/webapp/meeting-history.html]: Done!

File: prototype/src/main/webapp/meeting-history.html (hunk 3 above)
Comment (vivianho18) [reply prototype/src/main/webapp/meeting-history.html]: Fixed :)

**Pull request merged**